lib/ruby_lsp/tapioca/server_addon.rb

```diff
@@ -0,0 +1,36 @@
+# typed: false
+# frozen_string_literal: true
+
+begin
+  # The Tapioca add-on depends on the Rails add-on to add a runtime component to the runtime server. We can allow the
+  # add-on to work outside of a Rails context in the future, but that may require Tapioca spawning its own runtime
+  # server
+  require "ruby-lsp-rails"
+rescue LoadError
+  return
+end
+
+require "tapioca/internal"
```

Comment:
Now that the runtime functionality is all here, I think we can remove the internals require from the `addon.rb` file. We're not interacting with Tapioca there, only here.